assets/css/_sass/website/_shame.scss

```diff
@@ -5,4 +5,14 @@
   thead th {
     padding-top: 0;
   }
+}
+
+/* 
+* This will be removed when we move to version X.X (laura help me here) of the helix-ui code. The version currently used on the site does not have styling for the hxHeading-5 class. 
```

Comment:
@nimbinatus I was unclear what version of the Helix-ui toolkit we will need to have access to the new header class names. This comment is basically placeholder to explain why I was adding styling for the time being for H5's.

Reply:
h5 appeared in v0.3.0+; we'll be moving to the latest, which is currently v0.4.1